### PR TITLE
[develop] Trello-MfKoewMB. Currencies page

### PR DESCRIPTION
**Pull request by wildeastengineer**
[Trello-MfKoewMB](https://trello.com/c/MfKoewMB). Currencies page

### CODE REVIEW

File: src/components/CurrenciesPage/CurrenciesPage.jsx [hunk 1]
```diff
@@ -1,20 +1,163 @@
-import React, { Component } from 'react';
+// React
```
Comment (AndreySurzhan) [inline src/components/CurrenciesPage/CurrenciesPage.jsx]: @wildeastengineer I like how you separate imports

File: src/redux/actions/authActions.js [hunk 2]
```diff
@@ -35,6 +53,24 @@ function logInByEmailFailed(error) {
...
+        profileRep.getProfile()
+            .then((data) => {
+                dispatch(getUserProfileFinished(data));
+                dispatch(getProjectsList());
+            })
+            .catch((error) => {
+                dispatch(getUserProfileFailed(error));
+            });
+
+        return {};
```
Comment (AndreySurzhan) [inline src/redux/actions/authActions.js]: strange :) Is it expected? I understands nothing here but it looks suspicion

File: src/redux/reducers/authReducer.js [hunk 3]
```diff
@@ -11,20 +11,19 @@ import {
...
-    fetching: {
-        inProgress: false,
-        error: null
-    },
-    data: {
-        email: 'some@email.com',
-        name: 'User'
-    }
-}
+/**
```
Comment (AndreySurzhan) [inline src/redux/reducers/authReducer.js]: when are you going to uncomment it?

Comment (wildeastengineer) [inline src/redux/reducers/authReducer.js]: @AndreySurzhan I'm not going to uncomment it. It's just a example of state. But not sure is it necessary.

Comment (wildeastengineer) [inline src/redux/reducers/authReducer.js]: Let stays here for awhile  :smile:

File: src/redux/actions/authActions.js (hunk 2 above)
Comment (wildeastengineer) [inline src/redux/actions/authActions.js]: Yes, it's an extra thing. Thanks

**Comment by wildeastengineer** [comment]
@AndreySurzhan If you want we can discuss at the next meeting

**Pull request merged**